.github/workflows/test-and-build-workflow.yml

```diff
@@ -28,11 +28,11 @@ jobs:
       # job-scheduler
       - name: Build and Test
         run: |
-          ./gradlew build -Dopensearch.version=1.3.0-SNAPSHOT
```

Comment:
How about we remove `-Dopensearch.version=` from here and let it pickup the default?

Reply:
Good catch, I see that was already updated in `main`. Will update here.